app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -23,16 +23,18 @@ import org.mozilla.focus.autocomplete.UrlAutoCompleteFilter
 import org.mozilla.focus.telemetry.TelemetryWrapper
 import org.mozilla.focus.telemetry.UrlTextInputLocation
 import org.mozilla.focus.utils.OnUrlEnteredListener
+import org.mozilla.focus.tiles.HomeTile
+import org.mozilla.focus.tiles.CustomTilesAccessor
 
 private const val COL_COUNT = 5
+const val HOME_TILES_DIR = "defaults/"
```

Comment:
nit: private? What about moving the other keys here too?

nit: This is `DEFAULT_HOME_TILES_DIR`. I feel picky about this because we're reading from `assets`, which we can't write to.